tests/test_msbar_masses.py

```diff
@@ -88,22 +88,22 @@ def test_compute_msbar_mass_VFNS(self):
                     q2_to=m2_computed[nf - 3],
                 )
             )
-        np.testing.assert_allclose(m2_computed, m2_test, rtol=5e-3)
+        np.testing.assert_allclose(m2_computed, m2_test, rtol=3e-3)
 
     def test_errors(self):
 
         # test mass ordering
-        with pytest.raises(ValueError, match="do not preserve the correct ordering"):
-            theory_dict.update(
-                dict(
-                    mc=1.0,
-                    mb=1.00001,
-                    Qmc=1.0,
-                    Qmb=1.00001,
-                )
-            )
-            compute_msbar_mass(theory_dict)
-        with pytest.raises(ValueError, match="masses need to be sorted"):
+        # with pytest.raises(ValueError, match="do not preserve the correct ordering"):
```

Comment:
Why you're not testing this any longer?

Reply:
I'd propose to drop this addition check, see https://github.com/N3PDF/eko/pull/85#discussion_r803488102